docs/series/LAMP-server.md

```diff
@@ -0,0 +1,241 @@
+# Deploy a LAMP server
+
+The **LAMP** stack, comprising **L**inux, **A**pache, **M**ySQL, and **P**HP owes much of its popularity and versatility to its open-source nature.
+
+- **L**inux, the operating system at its core, embodies collaborative development, providing a stable foundation for web servers.
+
+- **A**pache, the iconic web server software, benefits from a community-driven approach, ensuring flexibility and reliability.
+
+- **M**ySQL, the relational database management system, democratizes access to robust database technology, thanks to its open-source ethos.
+
+- **P**HP, thrives on community contributions, facilitating dynamic and interactive web experiences.
+
+Together, the LAMP stack exemplifies the power of open-source collaboration, empowering developers worldwide to create scalable, cost-effective web applications that drive the modern internet.
+
+## Introduction
+
+For the sake of this guide we'll focus on a **LAMP** server that is composed of these four components:
+
+| Name | Description |
+|----------|----------|
+| (**L**) AlmaLinux    | The operating system, in this case AlmaLinux 8   |
+| **A**pache 2   | A trusted open-source web server software known for its reliability and scalability.  |
+| **M**ariaDB  | A popular open-source database system for managing structured data efficiently.   |
+| **p**hp-fpm   | A widely-used web scripting language known for its simplicity, flexibility, and strong community.   |
+
+### Requirements
+
+* A privileged user (*sudoer*) or access to the root user.
+* RHEL based OS, AlmaLinux 8 or 9.
+
+## First step - install Apache 2
+
+Apache is a project designed to develop and maintain an open source HTTP server for modern operating systems along with high security with efficient resource consumption and meeting HTTP standards.
+It first appeared in 1995 and since 1996 the most popular web server in the world.
+
+* **Install Apache 2:**
+    ```sh
+    sudo dnf install -y httpd
+    ```
+
+* **Allow traffic on port 80:**
+    ```sh
+    sudo firewall-cmd --add-port=80/tcp --permanent
+    sudo firewall-cmd --reload
+    ```
+
+* **Enable and start apache 2:**
+    ```sh
+    sudo systemctl enable httpd
+    sudo systemctl start httpd
+    ```
+
+You can test that Apache runs properly by accessing http://YOUR-IP (**YOUR-IP** being your ip address), you should get the following page:
+
+![AlmaLinux-Apache-Default-page](/images/HowTo-LAMP-server-1.png)
+
+## Second step - install MariaDB
+
+MariaDB is one of the major relational databases in the market, built by the developers of MySQL, ensuring to remain open-source, with a large part of the cloud utilizing it for data management purposes.
+
+
+* **To use a newer repo of MariaDB (To install a newer version) run the following command in your cmd:**
+    ```sh
+    curl -LsS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | sudo bash -s -- --mariadb-server-version=11.2
```

Comment:
This is a preference of mine, so make your own decision. Generally speaking, using the `curl .. | sudo bash` idiom is frowned upon due to the implications--potentially running random code from the internet. As such, I'd suggest something like this:

```bash
cat <<EOF>/etc/yum.repos.d/MariaDB.repo
# MariaDB 11.2 RedHatEnterpriseLinux repository list - created 2024-05-16 18:27 UTC
# https://mariadb.org/download/
[mariadb]
name = MariaDB
# rpm.mariadb.org is a dynamic mirror if your preferred mirror goes offline. See https://mariadb.org/mirrorbits/ for details.
# baseurl = [https://rpm.mariadb.org/11.2/rhel/$releasever/$basearch](https://rpm.mariadb.org/11.2/rhel/)
# baseurl = [https://rpm.mariadb.org/11.2/rhel/$releasever/$basearch](https://rpm.mariadb.org/11.2/rhel/)
module_hotfixes = 1
# gpgkey = https://rpm.mariadb.org/RPM-GPG-KEY-MariaDB
gpgkey = https://mirror.its.dal.ca/mariadb/yum/RPM-GPG-KEY-MariaDB
gpgcheck = 1
EOF
```

Reply:
^ My paste got goofed formatting wise. You'll want to copy the repo from their website: https://mariadb.org/download/?t=repo-config&d=Red+Hat+Enterprise+Linux+8&v=11.2&r_m=acorn

Also, I think the latest stable is `11.3` now.

Reply:
Thanks, I changed it based on your suggestion :smile:.